Amigolden/src/app/components/map/map.component.ts

```diff
@@ -31,54 +43,199 @@ export class MapComponent implements OnInit {
   withinMiles = 10;
   latitude = 40.7362942;
   longitude = -73.9921495;
+
   zoom: number;
-  address: string;
+  address = '';
+  town = '';
+  state = '';
   isCreating = false;
-  locationEntitiesMap = new Array<{location: Location, data: Array<any>}>();
+  i = 0;
+  locationEntitiesMap = new Array<{ location: Location; data: Array<any> }>();
   private geoCoder;
+  icon = {
+    url: './assets/images/maps-and-flags.svg',
+    scaledSize: {
+      width: 40,
+      height: 60,
+    },
+  };
+
+  @ViewChild('search', { static: false })
+  public search: IonInput;
+
+  @ViewChild(IonSlides, { static: false }) slides: IonSlides;
+
+  slideChanged() {
+    this.slides.getActiveIndex().then((index) => {
+      this.i = index;
+    });
+  }
+
+  // slider functions
 
-  @ViewChild('search', {static: false })
-  public searchElementRef: ElementRef;
+  slidePrev() {
+    this.slides.slidePrev();
+  }
+  slideNext() {
+    this.slides.slideNext();
+  }
+
+  //Flip
+  slideOptions = {
```

Comment:
please conform to the tslint.json, you should be able to see the error in the project if you're using VSCode. Declaration of instance field is not allowed after methods